log4j-docgen/src/test/resources/expected/processor/JavadocExample.adoc

```diff
@@ -0,0 +1,105 @@
+This paragraph has two sentences.
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+Vestibulum blandit dictum sem, ornare posuere lorem convallis sit amet.
+Sed dui augue, faucibus ut nisi id, mollis euismod nibh.
+Donec lobortis luctus viverra.
+In orci ante, pretium et fringilla at, sagittis nec justo.
+Cras finibus lorem vel volutpat interdum.
+Sed laoreet libero eros, ac cursus nibh dapibus vitae.
+Integer ante lorem, rhoncus at tortor vel, congue accumsan lorem.
+In hac habitasse platea dictumst.
+Nunc luctus ornare justo.
+Etiam ut metus id tortor dignissim semper.
+Nam turpis dui, aliquet nec enim et, faucibus accumsan dui.
+
+Aenean tincidunt elit id posuere mattis.
+Fusce bibendum sapien sed risus ultricies, non molestie erat volutpat.
+Donec nisi felis, egestas eu lobortis id, vulputate nec felis.
+In at dui euismod, blandit nulla et, accumsan elit.
+Proin id venenatis dui.
+Suspendisse sit amet est ut neque tincidunt venenatis.
+Donec bibendum quis velit fermentum porttitor.
+Maecenas faucibus, eros sit amet maximus malesuada, turpis neque bibendum justo, eu vehicula justo metus a ipsum.
+In at ullamcorper ipsum.
+Quisque in vehicula erat.
+Proin vitae suscipit dui, rutrum hendrerit augue.
+Curabitur finibus feugiat elit.
+
+. Item with a nested ordered list.
++
+.. First nested item.
+.. Second nested item.
+. Item with a nested unordered list.
++
+* Unordered list item.
+. Item with complex content
++
+Mauris suscipit velit nec ligula mattis, nec varius augue accumsan.
+Curabitur a dolor dui.
+Quisque congue facilisis est nec dignissim.
+Pellentesque egestas eleifend faucibus.
+Fusce imperdiet ligula a lectus fringilla varius.
+Sed malesuada porta vulputate.
+Sed vulputate purus nec nibh interdum convallis.
+Cras faucibus, dolor tempus lacinia vehicula, elit risus luctus libero, sed molestie nisi lorem sit amet enim.
+Integer vitae enim sagittis, malesuada lorem at, interdum tellus.
+Suspendisse potenti.
+Vestibulum ac nisi sit amet ex dictum suscipit.
+Nulla varius augue a velit tincidunt feugiat.
+Proin fringilla id leo ut dignissim.
+Vivamus eu tellus eget orci suscipit viverra.
+Donec sodales et arcu vel mollis.
++
+Praesent gravida auctor lectus quis interdum.
+Etiam semper mauris quis neque bibendum molestie.
+Maecenas a lacus nec risus pellentesque accumsan.
+Suspendisse dictum dui eleifend nibh facilisis, non consequat neque elementum.
+Donec scelerisque ultricies ipsum, pretium elementum ex pellentesque malesuada.
+Mauris egestas massa vitae sapien lobortis convallis.
+Donec feugiat, purus commodo consequat vehicula, dolor urna aliquam arcu, id rutrum quam tortor quis libero.
+Sed varius justo eget congue lacinia.
+
+* Item of an unordered list.
+
+== First section
+
+[cols="1,1",options="headers"]
+|===
+
+| Key
+| Value
+
+| A
+| 1
+
+| B
+| 2
+
+|===
+
+=== Subsection
+
+[source]
+----
+     private static final Logger logger = LogManager.getLogger();
```

Comment:
Can we remove the leading whitespace?

Reply:
That line is actually in a `<pre>` tag? Should we strip leading spaces there?

I could strip the same amount of leading whitespace from each row.

Reply:
Done: the leading space common to all the lines in the `<pre>` block is removed. Only relative indentation is left.